Bender.yml

```diff
@@ -100,9 +100,9 @@ sources:
 
   - target: all(synthesis, not(fpga))
     files:
-      - tech/sourcecode/tc_clk.sv
-      - tech/sourcecode/tc_sram.sv
-      - target/synth/carfield_synth_wrap.sv
+      - gf12/sourcecode/tc_clk.sv
+      - gf12/sourcecode/tc_sram.sv
+      - target/synth/src/carfield_synth_wrap.sv
```

Comment:
yes, I ran the synthesis trials in this branch and these path were wrong.

Reply:
I see, then i can restore them. I had the errors because they was pointing to "tech", but i cloned manually the gf12 dir so it was not named "tech".

I keep the changes for the last one "- target/synth/src/carfield_synth_wrap.sv" as the old one is wrong in both the cases.